### PR TITLE
Support miamiherald.com

### LINKED ISSUE #398: Add Miami Herald
Add miamiherald.com

For example
https://www.miamiherald.com/news/local/obituaries/article237255069.html

Thanks!

### PR DESCRIPTION
**Pull request by kimkha**
Resolves #398.

### CODE REVIEW

File: src/js/background.js [hunk 1]
```diff
@@ -26,6 +26,7 @@ const allowCookies = [
   'medium.com',
   'mercurynews.com',
   'mexiconewsdaily.com',
+  'miamiherald.com',
```
Comment (iamadamdev) [inline src/js/background.js:29]: In this case I don't think we need this here, just adding to googlebot is fine

File: src/js/background.js [hunk 2]
```diff
@@ -79,6 +80,7 @@ const removeCookies = [
...
```
Comment (iamadamdev) [inline src/js/background.js:83]: In this case I don't think we need this here, just adding to googlebot is fine

File: src/js/background.js (hunk 1 above)
Comment (kimkha) [reply src/js/background.js]: You're right.

Usually, I blindly copy from other site setting, this time is `mexiconewsdaily.com`. 😅

**Comment by ChiefMikeK** [comment]
Any reason why this **isn't** in latest version?  
https://github.com/iamadamdev/bypass-paywalls-chrome/commit/596d85fceac66a464cdc2d403e0ab2a7071b779e#diff-04c6e90faac2675aa89e2176d2eec7d8

**Comment by iamadamdev** [comment]
It will be included in the next release.

**Pull request merged**